packages/vite/src/node/cli.ts

```diff
@@ -151,6 +153,10 @@ cli
   })
   .option('-l, --logLevel <level>', `[string] info | warn | error | silent`)
   .option('--clearScreen', `[boolean] allow/disable clear screen when logging`)
+  .option(
+    '--configLoader <loader>',
+    `[string] use 'bundle' to bundle the config with esbuild or 'runner' to process it on the fly (default: bundle)`,
+  )
```

Comment:
Do we want to flag this experimental? I guess it's fine not to, but just to confirm.

Reply:
I think we can mark it as experimental since the environment API is experimental 😄

Reply:
Marked it like this:

>     `[string] use 'bundle' to bundle the config with esbuild or 'runner' (experimental) to process it on the fly (default: bundle)`,